LICENSE.txt

```diff
@@ -0,0 +1 @@
+see COPYING
```

Comment:
Again PyPI required the file, instead of duplicating I just referenced the actual license